### PR TITLE
11 port profile customization

### LINKED ISSUE #11: Port Profile Customization Page
- update user in db via server action

### PR DESCRIPTION
**Pull request by Hydrateurself**
closes #11

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
   "dependencies": {
     "@heroicons/react": "^2.1.5",
     "@hookform/resolvers": "^3.9.0",
+    "@lucia-auth/adapter-drizzle": "^1.0.7",
```
Comment (dnmct) [inline package.json]: why did you install this dependency? We are not using drizz;e

File: src/lib/types/user.ts
Comment (dnmct) [inline src/lib/types/user.ts:1]: Are you using this somewhere?
If yes this type can be constructed from the prisma types

**Review by dnmct — CHANGES_REQUESTED** [review]
Please also cleanup unused imports

File: package.json (hunk 1 above)
Comment (Hydrateurself) [reply package.json]: i installed it bc of a lucia tutorial. thought it would be standard for using it 🥸

**Review by dnmct — APPROVED** [review]

**Pull request merged**